test/hosting/NuGet.Config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="ilpkg" value="d:\temp\ilpackage" />
```

Comment:
I don't understand this change.  Where is "ilpkg" used, and how do we know this fixed path is valid?

Reply:
Oops.. this should not be here. This was local testing.

Reply:
Reverted.